tools/ci_build/build.py

```diff
@@ -875,6 +876,10 @@ def main():
     if args.build_csharp:
         args.build_shared_lib = True
 
+    if args.use_openvino == "VAD-F_FP32":
+        args.test = False
+        args.skip_onnx_tests = True
```

Comment:
setting args.tests = False should be sufficient. don't need to also set skip_onnx_tests.
can you also a comment as to why tests are disabled for VAD-F_FP32

Reply:
Cool. Will add this